
<launch>
    <node name= "ppm_reading" pkg="capstone2020" type="ppm_reader.py" />
    <node name= "ppm_sending" pkg="capstone2020" type="control_sender.py" />
    <node name= "test_gps_node" pkg="capstone2020" type="test_gps_node.py" />
    <node name= "test_control_node" pkg="capstone2020" type="test_control_2.py" output="screen" />
    <!--node name= "set_area_node" pkg="capstone2020" type="set_area.py" output="screen" /-->
    
</launch>